packages/contact-center/store/src/store.ts

```diff
@@ -24,6 +26,8 @@ class Store implements IStore {
     return this.cc.register().then((response: Profile) => {
       this.teams = response.teams;
```

Comment:
Does it make sense to copy the whole profile now that we are fetching more and more data ?

Reply:
Copying the whole profile will cause unnecessary rerender on each widgets.